bw-prep.js

```diff
@@ -27,12 +27,6 @@ const YAML = require('yaml');
   });
 
   var packages = [
```

Comment:
Great job! a big step forward. Please remove the codes added during the co-authoring chaos. ^^